Update timeout for windows PR validation

karthiknadig opened the pull request:
Looks like after updating to Jedi 0.16.0 we are running into timeout with Windows Single work-space runs. I am increasing the timeout temporarily until we figure out why.

DonJayamanne commented:
> Looks like after updating to Jedi 0.16.0 we are running in

This was always the case, since we refactored CI.
Previously the timeout was 120 or 180 (@karrtikr might remember).
Lol, we thought we fixed it, but we didn't (https://github.com/microsoft/vscode-python/issues/10293)

karrtikr commented:
@DonJayamanne Yes, but it almost always happens nowadays. Something may have triggered it.

karthiknadig commented:
We split the functional and single to run on separate machines, with #10293. Ian made the split. It was working better until we switched to Jedi 0.16.0 and we have data both from our users that switching to jedi 0.16.0 slowed down startup time. When we switch over to github actions we are going to have tests that run on specific are for PR and CI will run the whole thing.
